src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_bicep.py

```diff
@@ -19,20 +20,21 @@
 from azure.cli.core.azclierror import InvalidTemplateError
 from azure.cli.core.mock import DummyCli
 
-cli_ctx = DummyCli()
-
 
 class TestBicep(unittest.TestCase):
+    def setUp(self):
+        self.cli_ctx = DummyCli(random_config_dir=True)
```

Comment:
This contradicts the convention used in https://github.com/Azure/azure-cli/pull/25689 that `random_config_dir=True` is set in `super().__init__` invocation:

https://github.com/Azure/azure-cli/blob/b223dd9d4379d5c633a9f4dba4941ae791531126/src/azure-cli/azure/cli/command_modules/config/tests/latest/test_config.py#L16-L17

Reply:
Its superclass is `unittest.TestCase`, which does not has `random_config_dir` param.